internal/cli/interpreter.go

```diff
@@ -230,6 +249,43 @@ func (ee *ExecutionEnvironment) SubmitTransaction(ctx context.Context, result *E
 	return nil
 }
 
+func (ee *ExecutionEnvironment) createInsufficientRCMessage(ctx context.Context, result *ExecutionResult) error {
+	if ee.rcLimit.absolute {
+		rc, err := ee.RPCClient.GetAccountRc(ctx, ee.Key.AddressBytes())
+		if err != nil {
+			return err
+		}
+		if ee.rcLimit.value < rc {
+			decValue, err := util.SatoshiToDecimal(ee.rcLimit.value, cliutil.KoinPrecision)
+			if err != nil {
+				return err
+			}
+			decRc, err := util.SatoshiToDecimal(rc, cliutil.KoinPrecision)
+			if err != nil {
+				return err
+			}
+			result.AddErrorMessage(fmt.Sprintf("Current RC limit: %v, RC available: %v", decValue, decRc))
+			result.AddErrorMessage(fmt.Sprintf("Try using a higher RC limit. Example: rclimit %v", decRc))
```

Comment:
I don't think we should immediately recommend going max rclimit. That somewhat defeats the purpose of this issue.

I would do the min of the current rclimit * 2 and the account's max rc.